dss/config.py

```diff
@@ -4,23 +4,37 @@
 
 from .blobstore import BlobStore
 from .blobstore.s3 import S3BlobStore
+from .blobstore.gcs import GCSBlobStore
 from .hcablobstore import HCABlobStore
 from .hcablobstore.s3 import S3HCABlobStore
+from .hcablobstore.gcs import GCSHCABlobStore
 
 
 class Config(object):
     _S3_BUCKET = None  # type: str
+    _GS_BUCKET = None  # type: str
 
     @staticmethod
     def get_cloud_specific_handles(replica: str) -> typing.Tuple[
             BlobStore, HCABlobStore, str]:
+
+        handle: BlobStore
         if replica == 'aws':
             handle = S3BlobStore()
             return (
                 handle,
                 S3HCABlobStore(handle),
                 Config.get_s3_bucket()
             )
+        elif replica == 'gcs':
+            pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+            credentials = os.path.join(pkg_root, "gcs-credentials.json")
```

Comment:
I think we want to always use the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, no?